vision/getting-started/image_editing.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<p>Where did this come from? This should really be added to the Documentation. I couldn't find it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>The docs are a WIP. It will be added soon.</p>